index.js

```diff
@@ -0,0 +1,18 @@
+const hamMenu = document.querySelector('.ham-menu');
+const mobileMenu = document.querySelector('.mobile-menu');
+const closeMenu = document.querySelector('.close-icon');
+const mobileLinks = document.querySelectorAll('.mobile-link');
+
+hamMenu.addEventListener('click', () => {
+  mobileMenu.classList.toggle('active');
+});
+
+closeMenu.addEventListener('click', () => {
+  mobileMenu.classList.toggle('active');
+});
+
+mobileLinks.forEach((link) => {
+  link.addEventListener('click', () => {
+    mobileMenu.classList.toggle('active');
+  });
```

Comment:
- [ ] I can't see the hamburger button on the mobile version. Theses are the functionalities needed:
On mobile the hamburger menu button is visible.

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.
- When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

## View:
![image](https://user-images.githubusercontent.com/50990925/230917623-d82417fb-43ee-449c-84ec-6ef79ca2b842.png)

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/230917722-64e24cd8-042b-4340-a009-0f30aeb9666e.png)

Reply:
I have pushed some code after submitting for review. Can you check the latest code? or should I request for another review.

Reply:
Sorry for the late response. I can see it's approved. Good job 👏